test/LeanCode.IntegrationTests/docker/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env -S docker compose -f
```

Comment:
Works, but isn't really how the docker-compose should be used. Let's remove that.

Reply:
`docker compose` is not supposed to be used via `docker compose -f path/to/docker-compose.yml`? Why not? :upside_down_face:

Jokes aside, this is _very_ conveniant for me to use because those files live in a separate directory. I'd prefer to keep this just because you're not at all required to use it (and we already have useless shebangs in other places where they make even less sense to be).

Reply:
Hmm... That's a reasonable argument.

Food for thought - maybe it is worth to re-think how we approach dev env for testing here? This docker-compose approach was coined a loong time ago, when I haven't really had any major opinion about that. Maybe "just" moving it somewhere else, or keeping it as a long-lived env would be better? Or maybe using something like TestContainers?

Reply:
I'm not saying that we should ditch compose. I think that for this case there isn't really a better option (and for sure that shouldn't be full K8s cluster with TF & Tilt). But maybe there is a better way to structure things 
(e.g. use compose to run the databases, but run tests directly on the PC).